ts/hooks/useHeaderSecondLevel.tsx

```diff
@@ -127,6 +129,7 @@ export const useHeaderSecondLevel = ({
     const enableDiscreteTransitionProps =
       enableDiscreteTransition && animatedRef
         ? {
+            ignoreSafeAreaMargin: alertProps !== undefined,
```

Comment:
What do you think of using `!!alertProps` instead of `alertProps !== undefined`?